src/test/ui/parser/issue-65122-mac-invoc-in-mut-patterns.rs

```diff
@@ -0,0 +1,26 @@
+// Regression test; used to ICE with 'visit_mac disabled by default' due to a
```

Comment:
`// check-only`?

Reply:
There is no such annotation to my knowledge.